package.json

```diff
@@ -29,7 +26,7 @@
     "test-connectivity": "npm run test-connectivity --workspaces --if-present",
     "replace-package": "node scripts/replace-package.js",
     "test-nodedriver": "bash .evergreen/test-node-driver.sh",
-    "compile": "lerna run compile",
+    "compile": "npm run compile --workspaces --if-present",
```

Comment:
In anticipation of not being able to use the latest lerna in CI anymore (due to them dropping support for exotic platforms) I dealt with this here. We'll have to still tweak a few more commands to make sure we never use lerna in mongosh CI anymore, though. More PRs to follow :)